lib/easypost/services/batch.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# The Batch service allows you to perform operations on multiple Shipments at once.
+class EasyPost::Services::Batch < EasyPost::Services::Service
+  # Create a Batch.
+  def create(params = {})
+    wrapped_params = { batch: params }
+
+    @client.make_request(:post, 'batches', EasyPost::Models::Batch, wrapped_params)
```

Comment:
suggestion: let's defined a variable with the value `EasyPost::Models::Batch at the top of this class and then pass it into each of these, will help fix the bug below where we have bad copy/paste from other classes and DRY up this code.

This is true of all of them throughout the PR

Reply:
feedback accepted